internal/consensus/metrics.go

```diff
@@ -336,10 +394,40 @@ func (m *Metrics) MarkBlockGossipComplete() {
 	m.BlockGossipReceiveLatency.Observe(time.Since(m.blockGossipStart).Seconds())
 }
 
+func (m *Metrics) MarkProposalProcessed(appValid bool) {
+	status := "accepted"
+	if !appValid {
+		status = "rejected"
+	}
+	m.ProposalReceiveCount.With("status", status).Add(1)
+}
+
+func (m *Metrics) MarkVoteExtensionReceived(appValid bool) {
+	status := "accepted"
+	if !appValid {
+		status = "rejected"
+	}
+	m.VoteExtensionReceiveCount.With("status", status).Add(1)
+}
```

Comment:
Optional suggestion on legibility:
```suggestion
func (m *Metrics) MarkProposalProcessed(accepted bool) {
	status := "accepted"
	if !accepted {
		status = "rejected"
	}
	m.ProposalReceiveCount.With("status", status).Add(1)
}

func (m *Metrics) MarkVoteExtensionReceived(accepted bool) {
	status := "accepted"
	if !accepted {
		status = "rejected"
	}
	m.VoteExtensionReceiveCount.With("status", status).Add(1)
}
```

Might not be worth it, but if we have a lot of these it might be worth having a

```go
var acceptLabel = map[bool]string{false: "rejected", true: "accepted"}

// …
m.ProposalReceiveCount.With("status", acceptLabel[accepted]).Add(1)
```

but YMMV.